sqlglot/generator.py

```diff
@@ -4041,3 +4044,26 @@ def pad_sql(self, expression: exp.Pad) -> str:
     def summarize_sql(self, expression: exp.Summarize) -> str:
         table = " TABLE" if expression.args.get("table") else ""
         return f"SUMMARIZE{table} {self.sql(expression.this)}"
+
+    def explodinggenerateseries_sql(self, expression: exp.ExplodingGenerateSeries) -> str:
+        generate_series = exp.GenerateSeries(**expression.args)
+        generate_series_transform = self.TRANSFORMS.get(exp.GenerateSeries)
+
+        parent = expression.parent
+        if isinstance(parent, (exp.Alias, exp.TableAlias)):
+            parent = parent.parent
+
+        if generate_series_transform:
+            generate_series_sql = generate_series_transform(self, generate_series)
+        elif hasattr(self, "generateseries_sql"):
+            generate_series_sql = getattr(self, "generateseries_sql")(generate_series)
+        else:
+            generate_series_sql = self.function_fallback_sql(generate_series)
+
+        if self.SUPPORTS_EXPLODING_PROJECTIONS and not isinstance(parent, (exp.Table, exp.Unnest)):
+            return self.sql(exp.Unnest(expressions=[generate_series_sql]))
+
+        if isinstance(parent, exp.Select):
+            self.unsupported("GenerateSeries projection unnesting is not supported.")
```

Comment:
I don't think so, you can't do `UNNEST(GENERATE_SERIES(...))` in a projection, it's only valid when you use `GENERATE_SERIES` in a `FROM/JOIN` clause and if that's the case we won't hit this `unsupported` call because the parent won't be a `Select`.